code/modules/client/client_procs.dm

```diff
@@ -347,8 +357,8 @@
 	// 	if(config_legacy.aggressive_changelog)
 	// 		changelog_async()
 
-	// ensure asset cache is there
-	INVOKE_ASYNC(src, PROC_REF(warn_if_no_asset_cache_browser))
+	// run post-init 'lint'-like checks
+	on_new_hook_stability_checks()
```

Comment:
this is literally just everything that goes "if things aren't loaded yet load them"